IntelliNode/wrappers/OpenAIWrapper.js

```diff
@@ -145,6 +144,122 @@ class OpenAIWrapper {
       throw new Error(connHelper.getErrorMessage(error));
     }
   }
+
+  async uploadFile(filePath) {
+    try {
+      const url = `${this.API_BASE_URL}/v1/files`;
+      const boundary = '--------------------------' + Date.now().toString(16);
+      const headers = {
+        'Content-Type': `multipart/form-data; boundary=${boundary}`,
```

Comment:
The same comment about HTTP applies here, in the constructor axos initiated with the header and api key.

This function should utilize the existing axios instead of re initiating the connection details.